src/Docfx.App/Config/BuildJsonConfig.cs

```diff
@@ -18,34 +22,65 @@ internal class BuildJsonConfig
     [JsonIgnore]
     private Dictionary<string, GroupConfig> _groups;
 
+    /// <summary>
+    /// Contains all the files to generate documentation, including metadata yml files and conceptual md files.
+    /// </summary>
     [JsonProperty("content")]
     public FileMapping Content { get; set; }
 
+    /// <summary>
+    /// Contains all the resource files that conceptual and metadata files dependent on, e.g. image files.
+    /// </summary>
     [JsonProperty("resource")]
     public FileMapping Resource { get; set; }
 
+    /// <summary>
+    /// Contains all the conceptual files which contains yaml header with uid and is intended to override the existing metadata yml files.
+    /// </summary>
     [JsonProperty("overwrite")]
     public FileMapping Overwrite { get; set; }
 
+    /// <summary>
+    /// Specifies pairing that is used for folder redirection rules.
+    /// </summary>
     [JsonProperty("pairing")]
     public List<ContentPairingInfo> Pairing { get; set; }
 
+    /// <summary>
+    /// Specifies the tags of xrefmap.
+    /// </summary>
     [JsonProperty("xrefTags")]
     public ListWithStringFallback XrefTags { get; set; }
 
+    /// <summary>
+    /// Specifies the urls of xrefmap used by content files. Currently, it supports following scheme: http, https, ftp, file, embedded.
```

Comment:
It seems we only support HTTP/HTTPS:
https://github.com/dotnet/docfx/blob/main/src/Docfx.Build.Engine/DocumentBuildContext.cs#L301

```suggestion
    /// Specifies the urls of xrefmap used by content files. Supports local file path and HTTP/HTTPS urls.
```